docker-compose-cloud.yaml

```diff
@@ -5,18 +5,6 @@ services:
     # The QUICKPIZZA_IMAGE env. variable enables the use of a locally built image,
     # created with `docker build -t image_name .`
     image: ${QUICKPIZZA_IMAGE:-ghcr.io/grafana/quickpizza-local:latest}
-    logging:
-      driver: ${LOGGING_DRIVER:-loki}
-      options:
-        loki-url: "https://${GRAFANA_CLOUD_LOKI_USER}:${GRAFANA_CLOUD_TOKEN}@${GRAFANA_CLOUD_LOKI_HOST}/loki/api/v1/push"
```

Comment:
If we're forwarding this to the grafana cloud module, does this mean that we can get rid of the `GRAFANA_CLOUD_LOKI_USER` and `GRAFANA_CLOUD_LOKI_HOST` env vars in `.env.local`?

If they are only needed for local loki perhaps we can add a comment in `.env.local` then.

Reply:
Argh I forgot non-cloud again :see_no_evil:. I'll update the local docker compose as well to keep everything mostly the same